library/std/src/sys/thread_local/guard/key.rs

```diff
@@ -28,33 +47,55 @@ pub fn enable() {
             crate::rt::thread_cleanup();
         }
     }
+
+    enable_thread();
+    enable_process();
 }
 
 /// On platforms with key-based TLS, the system runs the destructors for us.
 /// We still have to make sure that [`crate::rt::thread_cleanup`] is called,
 /// however. This is done by defering the execution of a TLS destructor to
 /// the next round of destruction inside the TLS destructors.
+///
+/// POSIX systems do not run TLS destructors at process exit.
```

Comment:
Are they guaranteed to not run? Or are they just not guaranteed to run? Does the implementation deal gracefully with implementations that run them anyway?